examples/hello/bpTimeWriter/helloBPTimeWriter.py

```diff
@@ -22,18 +22,18 @@
 nx = myArray.size
 
 # ADIOS
-adios = ADIOS(comm, adiosDebugON)
+adios = adios2.ADIOS(comm, adios2.adiosDebugON)
 
 # IO
 bpIO = adios.DeclareIO("BPN2N")
 
 # Variables
 bpArray = bpIO.DefineVariable("bpArray", [size * nx], [rank * nx], [nx],
-                              adiosConstantDims)
+                              adios2.adiosConstantDims)
```

Comment:
I see you're using the module explicitly, I will change adiosConstantDims to ConstantDims in the glue code